### PR TITLE
fix: need to add in cash out codes for currency transaction activity …

### PR DESCRIPTION
**Pull request by mimy2001**
Cash out codes are missing for currency transaction activity detail type 

Screenshot 2024-01-04 at 5 08 49 PM

### CODE REVIEW

**Review by adamdecaf — APPROVED** [review]

**Comment by mimy2001** [comment]
@adamdecaf Hey Adam, i don't have any options to merge this, is that new? I can't remember last time if I got an approval I can just push the code straight in or is it due to the code coverage bot above?

**Comment by adamdecaf** [comment]
Did we give you commit access to the repo? Usually a Moov employee reviews the code and merges when it looks good.

**Comment by mimy2001** [comment]
ah ok :) i probably just remembered wrong, it's been a while since I've pushed a new fix. But I just put up another PR today.

**Pull request merged**
